src/tep_app.erl

```diff
@@ -6,10 +6,27 @@
 
 -behaviour(application).
 
+%%% Export these internal functions if EUnit is testing this module
+-ifdef(EUNIT).
+
+-export([get_config_argument/1]).
+
+-endif.
+
 -export([start/2, stop/1]).
 
-start(_StartType, _StartArgs) -> tep_sup:start_link().
+start(_StartType, _StartArgs) -> tep_sup:start_link(get_config_file_path()).
```

Comment:
That **doesn't** get formatted onto two lines? Kinda weird imho... I don't yet understand this formatter's preferences.

Reply:
Same here